network/server.go

```diff
@@ -211,7 +213,13 @@ func NewServer(logger hclog.Logger, config *Config) (*Server, error) {
 	srv.identity.setup()
 
 	// start gossip protocol
-	ps, err := pubsub.NewGossipSub(context.Background(), host)
+	bufferSize := int(config.PubSubBuffersSize)
+
+	ps, err := pubsub.NewGossipSub(
+		context.Background(),
+		host, pubsub.WithPeerOutboundQueueSize(bufferSize),
+		pubsub.WithValidateQueueSize(bufferSize),
+	)
```

Comment:
cc @Kourin1996 @lazartravica 

You're using this `bufferSize` value for the topic creation, **as well as** for initializing the outbound queue size for the peer, as well as the validation queue size, which is misleading.

Do you guys think this should be split into multiple flags?

Reply:
Resolving because we agreed internally these shouldn't be configurable yet